season-config.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<meta charset="UTF-8">
+<link rel="stylesheet" type="text/css" href="styles.css"> <!--css sheet-->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script> <!--jquery library-->
+<script src="script.js"></script> <!--js script-->
+<html>
+    <head>
+        <link rel="shortcut icon" href="images/club_logo.png" type="image/x-icon" />
+        <title>FRC 6321 Scouting</title>
+        <!--Importing fonts-->
+        <link href="https://fonts.googleapis.com/css?family=Ubuntu&display=swap" rel="stylesheet">
+    </head>
+    <body id="background" onload="loadSeasons()">
+        <button style = "float:left"type="button" onclick = "openNav()" id="buttons">Navigation Menu</button>
```

Comment:
If you use the `<button` element, you only need to specify the `type` if you're inside a `form` that's trying to assume the button is the submit.